packages/noya-state/src/selectors/pointSelectors.ts

```diff
@@ -233,6 +260,26 @@ export const moveSelectedPoints = (
         y: getNewValue(layer.frame.y, mode, delta.y),
       };
 
+      // if (draftLayer.frame.height === 0) {
```

Comment:
Do we still need this?

Reply:
I don't think so, not till we generalize to _any_ degenerate path instead of just a straight line with 2 points. We can remove it for now and come back to it